data_prep/graph_dataset.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, corpus):
         self._data.val_mask = val_mask
         self._data.test_mask = test_mask
 
+        print(self._data)
```

Comment:
This should be maybe gone?

Reply:
I didn't realise we actually want to merge this